### DISCUSSION TITLE
通过command line和通过code运行 paddleocr，同一张图片的运行结果不一致

### DISCUSSION THREAD

**Discussion by BoiceQian** [PaddlePaddle/PaddleOCR#13151] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：centos 
- 版本号/Version：Python 3.8.8  Paddle：2.1  PaddleOCR： 
- 问题相关组件/Related components：
- 运行指令/Command Code：
- by command code
  ```bash
   $ paddleocr --image_dir ./test_images/xx --lang=en
   ```
- by python code
  ```python
  from paddleocr import PaddleOCR
  ocr = PaddleOCR(use_angle_cls=True, lang='en',cls_model_dir=cls_model_dir, rec_char_dict_path=rec_char_dict_path,rec_model_dir=rec_model_dir,det_model_dir=det_model_dir,det_db_score_mode='slow')
  img_path = xx
  result = ocr.ocr(img_path, cls=True)
  for line in result:
      print(line)
  ```

- 完整报错/Complete Error Message：
    结果不一致，command line效果更好

**Comment by SWHL** [comment]
如果问题依然存在，请尝试main分支最新代码
